widget/view.controller.js

```diff
@@ -209,6 +248,10 @@
                     $scope.processingPicklist = false;
                     $scope.huntToolsMapping = response['hydra:member'][0].jSONValue;
                     $scope.threatHuntTools = Object.keys($scope.huntToolsMapping).sort();
+                    const index = $scope.threatHuntTools.indexOf("NIST National Vulnerability Database");
```

Comment:
NIST National Vulnerability Database" --> make it a const

Reply:
It will not going to change and by declaring it one more variable will used